src/LandingPage.js

```diff
@@ -7,8 +8,16 @@ export default class LandingPage extends Component {
         <h1>Photo site for Sallie and Jess</h1>
         <h4>Welcome to the photo sharing site for the wedding!</h4>
         <h5>Please log in or sign up to share your photos!</h5>
-        <button onClick={this.props.toLogIn}>Log In</button>
-        <button onClick={this.props.toSignUp}>Sign Up</button>
+        <button>
+          <Link className='littleLink' href='/login'>
```

Comment:
They class is so that I can attach CSS to all the links if I decide to do my own CSS.  I am not sure what you mean by defined.  Some kind of documentation?